packages/acquirer-backend/src/routes/userControllers/addUserByAdmin.ts

```diff
@@ -185,12 +182,12 @@ export async function addUser (req: AuthRequest, res: Response) {
       )
 
       // Generate Token using jwt
-      const token = jwt.sign({ id: newUser.id, email: newUser.email }, JWT_SECRET, { expiresIn: '1y' })
+      const token = jwt.sign({ id: newUser.id, email: newUser.email }, JWT_SECRET, { expiresIn: '1h' })
```

Comment:
does it make sense to make this time period configurable?

Reply:
yes. we can make it configurable via env

Reply:
but I felt like 1hr expire time was reasonable enough.. doesn't require much configuration.